engine/src/test/java/net/jqwik/api/EMailsTests.java

```diff
@@ -0,0 +1,228 @@
+package net.jqwik.api;
+
+import com.tngtech.archunit.thirdparty.com.google.common.net.*;
+
+import static net.jqwik.api.ArbitraryTestHelper.*;
+
+public class EMailsTests {
+
+	@Example
+	void containsAtSign(){
+		RandomGenerator<String> generator = getEmailGeneratorFromArbitrary();
+		assertAllGenerated(generator, (String value) -> value.contains("@"));
+	}
```

Comment:
Have you tried using a "real" property instead, like 

```
	@Property
	void containsAtSign(@ForAll("emails") String email) {
        Assertions.assertThat(email).contains("@");
	}
    @Provide
    Arbitrary<String> emails() {
        return Arbitraries.emails();
    }
```

Reply:
Only here? Or for all @Example ?

Reply:
> I wonder if the checks in the tests could be simplified by partially using regex matches.

Sure, but I like it more on this way. Is this a problem?

Reply:
Whatever you think is more readable.

Reply:
> Only here? Or for all `@Example ?`

Be consistent. I prefer properties whenever I can. In this case they seem a good fit to me.